README.md

```diff
@@ -190,6 +190,21 @@ import Imgix from "react-imgix";
 />;
 ```
 
+As of version 8.6.0, fixed image rendering will automatically append a `q` parameter matched against each `dpr` parameter when generating a fixed-size srcset. This behavior will respect any overriding `q` value passed in via `imgixParams` and can be disabled altogether with the new boolean property `disableQualityByDPR`.
```

Comment:
Can we add a sentence here explaining why we do this? I'm worried that this probably won't be obvious to most folks and they'll be left in confusion after reading this paragraph. Something like this, maybe?

> This technique is commonly used to compensate for the increased filesize of high-DPR images. Since high-DPR images are displayed a higher pixel density on devices, image quality can be lowered to reduce overall filesize without sacrificing perceived visual quality. For more information and examples of this technique in action, see [this blog post](https://blog.imgix.com/2016/03/30/dpr-quality).

Reply:
Totally, that's a great point.